IntegrationTests/Runner/Utils/ConsoleOutputInterceptor.swift

```diff
@@ -0,0 +1,65 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+#if DEBUG
+
+import UIKit
+import DatadogInternal
+import Datadog
+
+class ConsoleOutputInterceptor {
```

Comment:
Similar, AFAIK we don't use it in integration scenarios. Did a quick lookup - no reference. We don't need to copy this file 👍.

Reply:
🧹